gnomad_constraint/constraint/constraint_pipeline.py

```diff
@@ -0,0 +1,72 @@
+# noqa: D100
+
+import argparse
+import logging
+
+import hail as hl
+
+from gnomad.resources.grch37.gnomad import public_release
+from gnomad.resources.grch38.reference_data import vep_context
+from gnomad_constraint.resources.resource_utils import (
+    get_processed_ht_path,
+    get_logging_path,
+    context_ht_path,
```

Comment:
This context Table already include information from vep, coverage table, and methylation level table. Do we want to include the code that create this context Table? Or whoever who runs the pipeline will use the same context Table we provide every time.

Reply:
Which branch is context_ht_path defined in?

We need to update coverage/methylation so would be good to include it in the code